clients/web/src/components/warnings/brave-warning.tsx

```diff
@@ -0,0 +1,52 @@
+import { JSXElement, Show, Suspense } from "solid-js";
+import { Info } from "../icons/info";
+
+// The Brave browser exports a navigator function to determine whether we are using it.
+declare global {
+  interface Navigator {
+    brave?: {
+      isBrave: () => Promise<boolean>;
+    };
+  }
+}
+
+export function BraveWarning() {
+  const isBrave = (navigator.brave && navigator.brave.isBrave()) || false;
+
+  return (
+    <Suspense>
```

Comment:
You don't need suspense, since you're not using a resource.

Reply:
In this case the `navigator.brave.isBrave()' returns a Promise. Is my assumption correct that the `Suspense` will wait for it to be resolved?

Reply:
I see my mistake now. Removed the suspense :)

Reply:
It only ever returns a Promise in Brave, so there's no need to wait for it to resolve.